### DISCUSSION TITLE
Can we run argocd in fips enabled mode?

### DISCUSSION THREAD

**Discussion by chetanggg666** [argoproj/argo-cd#17324] [General]
Can we run argocd in fips-enabled mode?  if not what and all libraries and algorithms used for crypto and tls is at least fips enabled?

**Comment by aravindhkudiyarasan** [comment]
Do we have an update on this query ? We looking for a FIPS enabled GitOps tool to run in our GovCloud. Is Argo suitable for it ?

**Comment by randallweytens** [comment]
Also interested! Running into issues with using SSH since ArgoCD doesn't work with TLS passthrough for git repos that require PKI.
